docs/resources/resource_group_project.md

```diff
@@ -0,0 +1,46 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "oktapam_resource_group_project Resource - terraform-provider-oktapam"
+subcategory: ""
+description: |-
+  
+---
+
+# oktapam_resource_group_project (Resource)
+
+
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `name` (String) The human-readable name of the resource. Values are case-sensitive.
+- `resource_group` (String)
+
+### Optional
+
+- `account_discovery` (Boolean)
+- `create_server_users` (Boolean) If `true`, 'sftd' (ASA Server Agent) creates corresponding local (unix or windows) user accounts in the ASA Project's servers.
```

Comment:
Since this resource is unique to Okta PAM teams, I suggest you remove mention of ASA in those objects. I think it reads OK if you just drop it as a prefix for Projects and Server Agents (cc @isaacdurham-okta do you agree?)